2024-08-28 05:10:54 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "MIGRATION_MODEL" not found (this database is empty); SQL statement:
SELECT ID, VERSION FROM MIGRATION_MODEL ORDER BY UPDATE_TIME DESC [42104-214]
2024-08-28 05:10:56 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "DATABASECHANGELOG" not found (this database is empty); SQL statement:
SELECT COUNT(*) FROM PUBLIC.DATABASECHANGELOG [42104-214]
2024-08-28 05:10:58 jdbc[4]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "DATABASECHANGELOGLOCK" not found (this database is empty); SQL statement:
SELECT COUNT(*) FROM PUBLIC.DATABASECHANGELOGLOCK [42104-214]
2024-08-28 05:10:58 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Table "DATABASECHANGELOG" not found; SQL statement:
SELECT COUNT(*) FROM PUBLIC.DATABASECHANGELOG [42102-214]
2024-08-28 17:45:19 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1375)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:473)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:463)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:453)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:791)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:195)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:83)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:412)
	at io.quarkus.arc.Arc.shutdown(Arc.java:58)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:47)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:222)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:70)
	at org.keycloak.quarkus.runtime.KeycloakMain.start(KeycloakMain.java:102)
	at org.keycloak.quarkus.runtime.cli.command.AbstractStartCommand.run(AbstractStartCommand.java:37)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1939)
	at picocli.CommandLine.access$1300(CommandLine.java:145)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2358)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2352)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2314)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2316)
	at picocli.CommandLine.execute(CommandLine.java:2078)
	at org.keycloak.quarkus.runtime.cli.Picocli.parseAndRun(Picocli.java:93)
	at org.keycloak.quarkus.runtime.KeycloakMain.main(KeycloakMain.java:88)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at io.quarkus.bootstrap.runner.QuarkusEntryPoint.doRun(QuarkusEntryPoint.java:61)
	at io.quarkus.bootstrap.runner.QuarkusEntryPoint.main(QuarkusEntryPoint.java:32)
2024-08-28 17:45:19 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1375)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:473)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:463)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:453)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:791)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:195)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:83)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:412)
	at io.quarkus.arc.Arc.shutdown(Arc.java:58)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:47)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:222)
	at io.quarkus.runtime.Quarkus.run(Quarkus.java:70)
	at org.keycloak.quarkus.runtime.KeycloakMain.start(KeycloakMain.java:102)
	at org.keycloak.quarkus.runtime.cli.command.AbstractStartCommand.run(AbstractStartCommand.java:37)
	at picocli.CommandLine.executeUserObject(CommandLine.java:1939)
	at picocli.CommandLine.access$1300(CommandLine.java:145)
	at picocli.CommandLine$RunLast.executeUserObjectOfLastSubcommandWithSameParent(CommandLine.java:2358)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2352)
	at picocli.CommandLine$RunLast.handle(CommandLine.java:2314)
	at picocli.CommandLine$AbstractParseResultHandler.execute(CommandLine.java:2179)
	at picocli.CommandLine$RunLast.execute(CommandLine.java:2316)
	at picocli.CommandLine.execute(CommandLine.java:2078)
	at org.keycloak.quarkus.runtime.cli.Picocli.parseAndRun(Picocli.java:93)
	at org.keycloak.quarkus.runtime.KeycloakMain.main(KeycloakMain.java:88)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at io.quarkus.bootstrap.runner.QuarkusEntryPoint.doRun(QuarkusEntryPoint.java:61)
	at io.quarkus.bootstrap.runner.QuarkusEntryPoint.main(QuarkusEntryPoint.java:32)
2024-08-28 17:56:35 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1375)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:473)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:463)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:453)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:791)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:195)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:83)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:412)
	at io.quarkus.arc.Arc.shutdown(Arc.java:58)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:47)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager$ShutdownHookThread.run(ApplicationLifecycleManager.java:431)
2024-08-28 17:56:35 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1375)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:473)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:463)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:453)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:791)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:195)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:83)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:412)
	at io.quarkus.arc.Arc.shutdown(Arc.java:58)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:47)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager$ShutdownHookThread.run(ApplicationLifecycleManager.java:431)
2024-08-28 18:05:30 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1375)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:473)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:463)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:453)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:791)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:195)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:83)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:412)
	at io.quarkus.arc.Arc.shutdown(Arc.java:58)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:47)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager$ShutdownHookThread.run(ApplicationLifecycleManager.java:431)
2024-08-28 18:05:30 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1375)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:473)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:463)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:453)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:791)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:195)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:83)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:412)
	at io.quarkus.arc.Arc.shutdown(Arc.java:58)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:47)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager$ShutdownHookThread.run(ApplicationLifecycleManager.java:431)
2024-08-28 18:16:40 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1375)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:473)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:463)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:453)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:791)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:195)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:83)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:412)
	at io.quarkus.arc.Arc.shutdown(Arc.java:58)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:47)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager$ShutdownHookThread.run(ApplicationLifecycleManager.java:431)
2024-08-28 23:55:12 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1375)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:473)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:463)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:453)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:791)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:195)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:83)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:412)
	at io.quarkus.arc.Arc.shutdown(Arc.java:58)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:47)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager$ShutdownHookThread.run(ApplicationLifecycleManager.java:431)
2024-08-29 00:02:03 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1375)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:473)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:463)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:453)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:791)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:195)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:83)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:412)
	at io.quarkus.arc.Arc.shutdown(Arc.java:58)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:47)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager$ShutdownHookThread.run(ApplicationLifecycleManager.java:431)
2024-08-29 00:02:03 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1375)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:473)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:463)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:453)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:791)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:195)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:83)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:412)
	at io.quarkus.arc.Arc.shutdown(Arc.java:58)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:47)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager$ShutdownHookThread.run(ApplicationLifecycleManager.java:431)
2024-08-29 21:43:39 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1375)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:473)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:463)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:453)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:791)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:195)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:83)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:412)
	at io.quarkus.arc.Arc.shutdown(Arc.java:58)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:47)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager$ShutdownHookThread.run(ApplicationLifecycleManager.java:431)
2024-08-29 21:53:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1375)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:473)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:463)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:453)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:791)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:195)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:83)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:412)
	at io.quarkus.arc.Arc.shutdown(Arc.java:58)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:47)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager$ShutdownHookThread.run(ApplicationLifecycleManager.java:431)
2024-08-29 21:53:00 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1375)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:473)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:463)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:453)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:791)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:195)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:83)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:412)
	at io.quarkus.arc.Arc.shutdown(Arc.java:58)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:47)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager$ShutdownHookThread.run(ApplicationLifecycleManager.java:431)
2024-08-29 21:58:04 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1375)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:473)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:463)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:453)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:791)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:195)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:83)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:412)
	at io.quarkus.arc.Arc.shutdown(Arc.java:58)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:47)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager$ShutdownHookThread.run(ApplicationLifecycleManager.java:431)
2024-08-29 21:58:04 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1375)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:473)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:463)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:453)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:791)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:195)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:83)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:412)
	at io.quarkus.arc.Arc.shutdown(Arc.java:58)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:47)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager$ShutdownHookThread.run(ApplicationLifecycleManager.java:431)
2024-08-29 23:01:01 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1375)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:473)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:463)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:453)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:791)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:195)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:83)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:412)
	at io.quarkus.arc.Arc.shutdown(Arc.java:58)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:47)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager$ShutdownHookThread.run(ApplicationLifecycleManager.java:431)
2024-08-30 02:56:08 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: The database is open in exclusive mode; can not open additional connections [90135-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.engine.SessionLocal.checkSuspended(SessionLocal.java:1240)
	at org.h2.engine.SessionLocal.transitionToState(SessionLocal.java:1232)
	at org.h2.engine.SessionLocal.setCurrentCommand(SessionLocal.java:1210)
	at org.h2.engine.SessionLocal.endStatement(SessionLocal.java:1693)
	at org.h2.command.Command.executeUpdate(Command.java:291)
	at org.h2.jdbc.JdbcConnection.rollbackInternal(JdbcConnection.java:1391)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:468)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:453)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:791)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:195)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:83)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:412)
	at io.quarkus.arc.Arc.shutdown(Arc.java:58)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:47)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager$ShutdownHookThread.run(ApplicationLifecycleManager.java:431)
2024-08-30 02:56:08 jdbc[4]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: Database is already closed (to disable automatic closing at VM shutdown, add ";DB_CLOSE_ON_EXIT=FALSE" to the db URL) [90121-214]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:678)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:477)
	at org.h2.message.DbException.get(DbException.java:223)
	at org.h2.message.DbException.get(DbException.java:199)
	at org.h2.message.DbException.get(DbException.java:188)
	at org.h2.jdbc.JdbcConnection.checkClosed(JdbcConnection.java:1375)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.checkClosed(JdbcXAConnection.java:473)
	at org.h2.jdbc.JdbcConnection.rollback(JdbcConnection.java:463)
	at org.h2.jdbcx.JdbcXAConnection$PooledJdbcConnection.close(JdbcXAConnection.java:453)
	at org.h2.jdbcx.JdbcXAConnection.close(JdbcXAConnection.java:76)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:185)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:791)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:195)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:83)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:21)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:96)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:412)
	at io.quarkus.arc.Arc.shutdown(Arc.java:58)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:47)
	at io.quarkus.runtime.StartupContext.runAllInReverseOrder(StartupContext.java:84)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:73)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.ApplicationLifecycleManager$ShutdownHookThread.run(ApplicationLifecycleManager.java:431)
